third-party/bazel/repos.bzl

```diff
@@ -21,8 +21,16 @@ def everest_core_repos():
     maybe(
         http_archive,
         name = "everest-framework",
-        url = "https://github.com/golovasteek/everest-framework/archive/e75752db299940219b2fae2cfddc23482129f921.tar.gz",
-        strip_prefix = "everest-framework-e75752db299940219b2fae2cfddc23482129f921",
+        url = "https://github.com/qwello/everest-framework/archive/2306559764824a61406da6641985517e3b61f193.tar.gz",
+        sha256 = "d1b9052558dacd383d47cc79e433d2df45beec0c06bf204c5570125a5c10cbcc",
+        strip_prefix = "everest-framework-2306559764824a61406da6641985517e3b61f193",
+    )
+
+    maybe(
+        http_archive,
+        name = "everest-utils",
+        url = "https://github.com/Qwello/everest-utils/archive/8d8c1b7172d114a6f4a0fc5f4d6e2d8ae1a0da82.tar.gz",
+        strip_prefix = "everest-utils-8d8c1b7172d114a6f4a0fc5f4d6e2d8ae1a0da82",
```

Comment:
git_repository? Then with just one commit it is a "reproduceable" setup.

With http_archive, you need to do strip_prefix, and generally you need sha256 for caching and repruceability.